src/application.rs

```diff
@@ -0,0 +1,27 @@
+use Application;
+use ffi;
+use glib::ApplicationFlags;
+use glib::translate::*;
+use rt;
+
+impl Application {
+    #[cfg(feature = "3.6")]
+    pub fn new(application_id: Option<&str>, flags: ApplicationFlags) -> Result<Application, ()> {
+        skip_assert_initialized!();
```

Comment:
"And if you are using GtkApplication, you don't have to call any of the initialization functions either; the “startup” handler does it for you.".